packages/near-api-js/test/wallet-account.test.js

```diff
@@ -154,12 +155,63 @@ it('can complete sign in', async () => {
     await walletConnection._completeSignInWithAccessKey();
 
     expect(await keyStore.getKey('networkId', 'near.account')).toEqual(keyPair);
-    expect(localStorage.getItem('contractId_wallet_auth_key'));
+    expect(localStorage.getItem('contractId_wallet_auth_key')).toBeTruthy();
     expect(history.slice(1)).toEqual([
         [{}, 'documentTitle', 'http://example.com/location']
     ]);
 });
 
+it('can get the default appKeyPrefix', async () => {
+    nearFakeWithoutContract = {
+        config: {
+            networkId: 'networkId',
+            walletUrl: 'http://example.com/wallet',
+        },
+        connection: {
+            networkId: 'networkId',
+            signer: new nearApi.InMemorySigner(keyStore)
+        },
+        account() {
+            return {
+                state() { }
+            };
+        }
+    };
```

Comment:
Can we do this in `beforeEach` just like `nearFake`? Also this is more concise:
```suggestion
    nearFakeWithoutContract = {...nearFake, config: {...nearFake.config, contractName: undefined}};
```